src/checkUsageCalc.ts

```diff
@@ -0,0 +1,63 @@
+import { CheckType } from './types';
+
+interface ActiveSeriesParams {
+  probeCount: number;
+  checkType: CheckType;
+  frequencySeconds: number;
+}
+
+interface UsageValues {
+  checksPerMonth: number;
+  activeSeries: number;
+  logsGbPerMonth: number;
+}
+
+enum CheckSeries {
+  PING = 76,
+  HTTP = 128,
+  DNS = 79,
+  TCP = 59,
```

Comment:
**Question:** probably a dumb question, but what do these numbers mean exactly?

Reply:
These numbers are the number of active series generated per check. Different types of checks generate different numbers of metrics

Reply:
Thanks for the explanation Nathan 👍